solr/core/src/java/org/apache/solr/handler/admin/LoggingHandler.java

```diff
@@ -47,23 +47,16 @@
 public class LoggingHandler extends RequestHandlerBase implements SolrCoreAware {
```

Comment:
```suggestion
public class LoggingHandler extends RequestHandlerBase {
```

maybe since `inform(SolrCore core)` is becoming a no-op or comment w.r.t. why `inform` is needed albeit as a no-op.